### ISSUE TITLE
Comand Pull

### ISSUE THREAD

**Issue by engBrunoSales** [FredrikNoren/ungit#1490]
Create comand Pull

**Comment by wmertens** [comment]
not quite - pull can also mean merging and handling conflicts. It would be nice if pull showed a nice merge arrow on hover.

But personally I never pull, I only fetch and move or rebase, so I don't care about this feature ;)

**Comment by jung-kim** [comment]
That's absolutely right, what I propose above is simple "move" which has no consideration for reconciling diffs.  It's rather just accepting one change over another.  

But "pull" in those case can be accomplished by existing merge operation so I would still maintain that there are alternatives.  And I think this alternative are better as operations are explicit in it's purpose unlike "pull" that have different meanings in different context.

**Comment by bes-internal** [comment]
for history: now switching to a remote branch does fast-forward automatically. Like in @jung-kim gif but without move mouse action; this is alse works for any overscreen long history) I think this removes 80% of the cases from this question on adding a pull button.
